help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,117 @@
+
+![Authentication screenshot](../../assets/catalog/analytics/gainsight_px/auth-screen.png)
+* **[!UICONTROL Password]**: The password used to login to [!DNL Gainsight PX](https://app.aptrinsic.com)
+* **[!UICONTROL Client ID]**: The Gainsight PX subscription ID on the [Company Details page](https://app.aptrinsic.com/settings/subscription)
+* **[!UICONTROL Client secret]**: The OAuth secret generated at the bottom of the [Company Details page](https://app.aptrinsic.com/settings/subscription) in the [!DNL Gainsight PX] UI.
+* **[!UICONTROL Username]**: The email used to log in to the [!DNL Gainsight PX](https://app.aptrinsic.com) UI
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![Destination details screen in the Experience Platform user interface showing how to fill in the Name and Description fields](../../assets/catalog/analytics/gainsight_px/destination_details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+>
+>* To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map identities {#map}
+
+Use the mapping step to configure the relationship between a identity value within the Experience Platform that matches the value passed into the Gainsight PX system during the identify step.  There only restriction is that the field is of type string and the value for a particular user match exactly the value used during the Gainsight PX identify step.   The only choice on the target side of the mapping is IDENTIFY_ID.
```

Comment:
The explanations around mapping are **extremely** confusing. I read this multiple times and I still don't understand what I should be mapping to what.

In addition to that, there are some typos and phrasing issues:

- "a identity" --> "an identity"


"identity value within the Experience Platform that matches the value passed into the Gainsight PX system during the identify step."
- What is the identify step? Where does that happen? Where does that value get passed into Gainsight _from_?


"There only restriction is that the field is of type string and the value for a particular user match exactly the value used during the Gainsight PX identify step."

- What do you mean by "there"? 
- Are you talking about the source field here? If so, please be specific.


"The only choice on the target side of the mapping is IDENTIFY_ID."
- If that is the case, what's the point of the whole explanation about a _"value passed into the PX system during the identify step"?_.

Please rewrite this section in a clearer manner, similar to the section used by this destination: https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/personalization/pega.html?lang=en#mapping-example

Your mapping section should answer these questions:

- What source field **can** I map?
- What target field **can** I use?
- Are there any mandatory mappings that I must add?

Reply:
Added new example
Simplified description